docs/getting_started.md

```diff
@@ -0,0 +1,99 @@
+# Getting started
+
+## Tools
+
+You'll need the following:
+
+* [node](https://nodejs.org/en/download/)
+
+Install them using your favorite method (`homebrew`, etc).
+
+## Repo
+
+First, [fork the repository](https://github.com/covidatlas/li.git) so you're ready to contribute back.
+
+### 1. Clone, init submodules, and add upstream
+
+Replace `yourusername` below with your Github username:
+
+```
+git clone git@github.com:yourusername/coronadatascraper.git
+cd li
+git remote add upstream git@github.com:covidatlas/li.git
+```
+
+### 2. Install dependencies
+
+```
+npm install
+```
+
+If you get an error message saying you have an incompatible version of
+`node`, you may need to change version.  You can use `n` to change
+node versions: [install](https://www.npmjs.com/package/n) it and run
+`n lts`.
+
+### Run the server
+
+Start the server in one window:
+
+```
+$ npm run start
+```
+
+The first time you run this will take a minute as it downloads dependencies.  At the end you'll see "Sandbox Started in 58873ms ... http://localhost:3333".
+
+Browse to http://localhost:3333/ to see the site.
+
+### Run a sample scraper
```

Comment:
`source` not `scraper` – we can begin retiring that term immediately

Reply:
Also, I'm not sure what a sample scraper would be? Should just have them run a simple one like SF?

Reply:
Yes, something easy, that's guaranteed to work, if guarantees are possible.

Reply:
I've added `nyt` as the sample.